bank-sdk/example-app/src/main/java/net/gini/android/bank/sdk/exampleapp/ui/CaptureFlowHostActivity.kt

```diff
@@ -20,8 +19,13 @@ class CaptureFlowHostActivity : AppCompatActivity() {
 
         if (savedInstanceState == null) {
             val openWithIntent = IntentCompat.getParcelableExtra(intent, EXTRA_IN_OPEN_WITH_INTENT, Intent::class.java)
-            if (openWithIntent != null) {
-                startBankSDKForOpenWith(openWithIntent)
+            val openDocument = IntentCompat.getParcelableExtra(intent, EXTRA_IN_OPEN_WITH_DOCUMENT, Document::class.java)
+            if (openDocument != null) {
+                startBankSDKForDocument(openDocument)
+            } else if (openWithIntent != null) {
+                intent.getParcelableExtra(EXTRA_IN_OPEN_WITH_DOCUMENT, Document::class.java)?.let {
+                    startBankSDKForDocument(it)
+                }
```

Comment:
@danicretu I think here we should directly use the `openWithIntent`:
```suggestion
                startBankSDKForOpenWith(openWithIntent)
```

Reply:
Fixed, thanks for catching this.